src/doc/src/reference/build-scripts.md

```diff
@@ -169,6 +169,30 @@ The optional `KIND` may be one of `dylib`, `static`, or `framework`. See the
 [option-link]: ../../rustc/command-line-arguments.md#option-l-link-lib
 [FFI]: ../../nomicon/ffi.md
 
+
+<a id="rustc-link-arg-tests"></a>
+#### `cargo:rustc-link-arg-tests=FLAG`
+
+The `rustc-link-arg-tests` instruction tells Cargo to pass the [`-C
+link-arg=FLAG` option][link-arg] to the compiler, but only when building a
+tests target.
+
+[link-arg]: ../../rustc/codegen-options/index.md#link-arg
```

Comment:
I think this `[link-arg]` definition can be removed. There only needs to be one in the file (Markdown only looks at one of them, ignoring the rest).  Probably can remove the other duplicates, too.